components/director/internal/domain/scenarioassignment/engine_service.go

```diff
@@ -0,0 +1,30 @@
+package scenarioassignment
+
+import "github.com/kyma-incubator/compass/components/director/internal/model"
+
+type assignmentEngineService struct{}
```

Comment:
package is called `scenarioassignment`, your type `assignmentEngineService`, maybe we can rename it to just `engine`?